sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -550,3 +472,120 @@ directive:
     where: $.definitions.HealthcareAssertion
     transform: $.description = "An object that describes metadata about the healthcare entity such as whether it is hypothetical or conditional.";
 ```
+
+## JS customizations
+
+### Task renames
+
+```yaml
+directive:
+  - from: swagger-document
```

Comment:
Tip: you can avoid so much nesting. YAML arrays can start on the same line as their parent element e.g.:

```yaml
directive:
- from: swagger-document
  where: ...
- from: swagger-document
  # ...
```